packages/advanced-logic/test/extensions/payment-network/any-to-erc20-proxy.test.ts

```diff
@@ -64,89 +64,92 @@ describe('extensions/payment-network/erc20/any-to-erc20-fee-proxy-contract', ()
     });
 
     it('cannot createCreationAction with payment address not an ethereum address', () => {
-      // 'must throw'
       expect(() => {
-        anyToErc20Proxy.createCreationAction({
+        new anyToErc20ProxyPaymentNetwork().createCreationAction({
           paymentAddress: 'not an ethereum address',
           refundAddress: '0x0000000000000000000000000000000000000002',
           salt: 'ea3bc7caf64110ca',
+          network: 'private',
+          acceptedTokens: [DataConversionERC20FeeCreate.tokenAddress],
```

Comment:
In order to focus on one error, we need to make everything else correct. Network & acceptedTokens are mandatory. (same comment for many tests below)

Reply:
very good point!